ui/components/MesheryAdapterPlayComponent.js

```diff
@@ -232,6 +232,7 @@ class MesheryAdapterPlayComponent extends React.Component {
     subscribeAddonStatusEvents(self.setAddonsState, variables)
 
     fetchAvailableAddons(variables)
+      .toPrmise()
```

Comment:
@dhruv0000 typo... Im fixing in my upcoming PR. // @leecalcote

Reply:
Raised another PR for the fix. [#2564](https://github.com/layer5io/meshery/pull/2564)